tests/unit/ui/components/TestRating.js

```diff
@@ -175,6 +175,19 @@ describe('ui/components/Rating', () => {
     expect(findDOMNode(root).textContent).toEqual('Rated 2 out of 5');
   });
 
+  it('renders star button with id in ratings', () => {
+    const root = render();
+    const elementKeys = Object.keys(root.ratingElements);
```

Comment:
This is a bit of a roundabout way to test for IDs, and there are helpers we have to look for IDs that would make this clearer.

You should be able to simply verify the IDs exist in one test (one should already exist?) and then use `expect(findDOMNode(root).querySelector('#Rating-ID')).toHaveLength(0)` or something similar.

Reply:
Can I use this ? 
``
expect(findDOMNode(root).querySelector('.Rating-choice').id).toHaveLength(0); 
``
But I have doubt that it is checking id for only the first Star.
I also thought of an alternate way like this

``
[1, 2, 3, 4, 5].forEach((rating) => {
        expect(root.ratingElements[rating].id).toHaveLength(0);
});
``

What should I use ?

Reply:
Sorry for my late reply, but unfortunately those tests aren't getting at what we're looking to check for. I don't think I'll have much time to mentor this issue further, but thanks for your efforts here 👍

Reply:
So should I close this PR ?

Reply:
It might be best to close it if this hasn't moved forward, but if you'd like to work on another issue there are more issues marked as contributor-friendly. Sorry about this one getting stalled.

Reply:
Okay No Worries. Will close this PR and try to work on some other. Thanks :)